src/frontend/pages/CartScreen/components/Cartdetails.js

```diff
@@ -24,7 +43,7 @@ const Cartdetails = () => {
         <div className="details">
           <div className="details__content">
             <h3>Total MRP</h3>
-            <h3>Rs 4000</h3>
+            <h3>Rs {totalPriceofitems - totalPriceofitems * (8 / 100)}</h3>
```

Comment:
We can take out this operation and put it inside a function ( helper ).

Reply:
got it.thanks